index.yaml

```diff
@@ -856,39 +856,17 @@ entries:
       catalog.cattle.io/ui-component: rancher-cis-benchmark
     apiVersion: v1
     appVersion: v1.0.5
-    created: "2021-06-16T21:05:17.237837-07:00"
+    created: "2021-06-17T22:33:25.928724-07:00"
     description: The cis-operator enables running CIS benchmark security scans on
       a kubernetes cluster
-    digest: 78201040c5b18951c71605fc2dbdf3a2265bfedc24d3587aea9df4b1671e3ad6
+    digest: 894bb5a1411b10340fca01b36fee412d81cd65ebf681fcf72d0be274774ddbc0
     icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
     keywords:
     - security
     name: rancher-cis-benchmark
     urls:
-    - assets/rancher-cis-benchmark/rancher-cis-benchmark-100.0.0.tgz
-    version: 100.0.0
-  - annotations:
-      catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
-      catalog.cattle.io/certified: rancher
-      catalog.cattle.io/display-name: CIS Benchmark
-      catalog.cattle.io/namespace: cis-operator-system
-      catalog.cattle.io/os: linux
-      catalog.cattle.io/provides-gvr: cis.cattle.io.clusterscans/v1
-      catalog.cattle.io/release-name: rancher-cis-benchmark
-      catalog.cattle.io/ui-component: rancher-cis-benchmark
-    apiVersion: v1
-    appVersion: v1.0.5
-    created: "2021-06-16T21:05:17.236176-07:00"
-    description: The cis-operator enables running CIS benchmark security scans on
-      a kubernetes cluster
-    digest: bc8c801eeac9fbbb60a9be9a72025f6df4f1835dc5cf80c18020a9462c1623af
-    icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
-    keywords:
-    - security
-    name: rancher-cis-benchmark
-    urls:
-    - assets/rancher-cis-benchmark/rancher-cis-benchmark-1.0.600.tgz
-    version: 1.0.600
```

Comment:
rancher-cis-benchmark 1.0.600 is  generated by the `make charts` command with chart-build-scripts 0.2.0, and it is not cleaned in the this [PR](https://github.com/rancher/charts/pull/1247) 

it is identical to 100.0.0 which now is 2.0.0 

proof:
```
> diff -r charts/rancher-cis-benchmark/rancher-cis-benchmark/1.0.600  charts/rancher-cis-benchmark/rancher-cis-benchmark/100.0.0 
diff -r charts/rancher-cis-benchmark/rancher-cis-benchmark/1.0.600/Chart.yaml charts/rancher-cis-benchmark/rancher-cis-benchmark/100.0.0/Chart.yaml
18c18
< version: 1.0.600
---
> version: 100.0.0
```